[CTOR-438] [Doc] [CommunityPR] Infor ion grid restapi (#3195)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Infor-Ion-Grid-Restapi-custom" label="App-Infor-Ion-Grid-Restapi-custom">

| Service Alias      | Service Template                              | Service Description                             | Discovery  |
|:-------------------|:----------------------------------------------|:------------------------------------------------|:----------:|
| Application-Status | App-Infor-Ion-Grid-Restapi-Application-custom | Check Infor ION Grid applications status        | X          |
| Node               | App-Infor-Ion-Grid-Restapi-Node-custom        | Check Infor ION Grid node status and statistics | X          |

> The services listed above are created automatically when the **App-Infor-Ion-Grid-Restapi-custom** host template is used.

> If **Discovery** is checked, it means a service discovery rule exists for this service template.

</TabItem>
</Tabs>

### Discovery rules

#### Service discovery

| Rule name                                   | Description |
|:--------------------------------------------|:------------|

> | App-Infor-Ion-Grid-Restapi-Application-Name | Discover anypoint of application Infor ION Grid and monitor their status |

Reviewer: ```suggestion
| App-Infor-Ion-Grid-Restapi-Application-Name | Discover all application points for Infor ION Grid and monitor their status |
```

Reviewer: what is a "point d'application"?

More information about discovering services automatically is available on the [dedicated page](/docs/monitoring/discovery/services-discovery)
and in the [following chapter](/docs/monitoring/discovery/services-discovery/#discovery-rules).

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics linked to each service.

<Tabs groupId="sync">
<TabItem value="Application-Status" label="Application-Status">

| Metric name                       | Unit  |
|:----------------------------------|:------|
| ion.grid.applications.total.count | count |
| *applications*#status             | N/A   |

</TabItem>
<TabItem value="Node" label="Node">

| Metric name                        | Unit  |
|:-----------------------------------|:------|
| *nodes*#status                     | N/A   |
| *nodes*#node.log.error.count       | count |
| *nodes*#node.log.warning.count     | count |
| *nodes*#node.uptime.seconds        | s     |
| *nodes*#node.cpu.usage.percentage  | %     |
| *nodes*#node.heap.usage.percentage | %     |

</TabItem>
</Tabs>

## Prerequisites

> *Specify prerequisites that are relevant. You may want to just provide a link\n\

Reviewer: Prerequisites

| CRITICALUPTIME        |Set critical thresholds on uptime                                                                                                                                    |                        |             |
| EXTRAOPTIONS          | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                  |                        |             |

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
is able to monitor a resource using a command like this one (replace the sample values by yours):

> ```bash

Reviewer: FR and EN must be the same
